packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -418,7 +418,14 @@ class ApplicationCommand extends Base {
         command.descriptionLocalizations ?? command.description_localizations ?? {},
         this.descriptionLocalizations ?? {},
       ) ||
-      !isEqual(command.integrationTypes ?? command.integration_types ?? [], this.integrationTypes ?? []) ||
+      // [0] is the default value sent by Discord
+      !isEqual(
+        command.integrationTypes ??
+          command.integration_types ??
+          this.client.application.integrationTypesConfig?.keys() ?? [ApplicationIntegrationType.GuildInstall],
```

Comment:
There is no `Object.prototype.keys()`. If this is to be kept, you have to use `Object.keys()` instead

Reply:
Yeah mb didn't realize this was a regular object, will change